### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by maharjanaman**
Separate system module from rubix os

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+	golang.org/x/sys v0.6.0 // indirect
+	golang.org/x/text v0.8.0 // indirect
+	google.golang.org/genproto v0.0.0-20230110181048-76db0878b65f // indirect
+	google.golang.org/grpc v1.54.1 // indirect
+	google.golang.org/protobuf v1.28.1 // indirect
+	gorm.io/driver/mysql v1.3.2 // indirect
+	gorm.io/gorm v1.23.2 // indirect
+)
+
+replace github.com/NubeIO/rubix-os => /Users/i8e4/CodeDuo/NubeIO/rubix-os
```
Comment (RaiBnod) [inline go.mod]: Put this line at first, so we get the first insight about the package replacement.

File: logger/logger.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package logger
+
+import "github.com/hashicorp/go-hclog"
+
+func SetLogger(logLevel string) {
+	hclog.SetDefault(hclog.New(&hclog.LoggerOptions{
+		Name:  "module-core-system",
+		Level: hclog.LevelFromString(logLevel),
+	}))
+}
```
Comment (RaiBnod) [inline logger/logger.go]: Implement this `logLevel` on the basis of config.

File: pkg/schedule.go [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+		if err != nil {
+			log.Error(fmt.Sprintf("Schedule Checks: issue on exceptionResult %s", err.Error()))
+		} else {
+			log.Info(fmt.Sprintf("Schedule Checks: exception schedule: %s  is-active: %t", exceptionResult.Name, exceptionResult.IsActive))
+		}
+		if exceptionResult.CheckIfEmpty() {
+			log.Info(fmt.Sprintf("Schedule Checks: exception schedule is empty: %s", exceptionResult.Name))
+		}
+		log.Info(fmt.Sprintf("Schedule Checks: exceptionResult: %+v", exceptionResult))
+
```
Comment (RaiBnod) [inline pkg/schedule.go]: These `log.Info`s needs to be in `log.Debug` level. It's just printing so many logs. Compare with the existing plugin.

**Review by RaiBnod — CHANGES_REQUESTED** [review]
See those change requests.

File: pkg/schedule.go (hunk 3 above)
Comment (maharjanaman) [reply pkg/schedule.go]: So we just change `Info` to `Debug` or remove it all together?

Comment (RaiBnod) [reply pkg/schedule.go]: Change into `Debug` and that level of the log needs to be customized from the config.

**Pull request merged**